v3.0/how-to/get-started/local-cluster/install-from-dbdeployer.md

```diff
@@ -29,31 +29,29 @@ Similar to [Homebrew](/how-to/get-started/local-cluster/install-from-homebrew.md
 
 After <a href="https://github.com/datacharmer/dbdeployer">installing DBdeployer</a>, install a MySQL 5.7 client:
 <pre>
-curl -LO https://dev.mysql.com/get/Downloads/MySQL-5.7/mysql-5.7.25-macos10.14-x86_64.tar.gz
-dbdeployer unpack mysql-5.7.25-macos10.14-x86_64.tar.gz
+dbdeployer downloads get-unpack --overwrite --delete-after-unpack mysql-5.7.26-macos10.14-x86_64.tar.gz
 </pre>
 
 Install TiDB using the installed MySQL 5.7 client:
 <pre>
 curl -O https://download.pingcap.org/tidb-master-darwin-amd64.tar.gz
```

Comment:
@zhouqiang-cl @kolbe It looks like we don't have a darwin 3.0 tidb-server only package. Can we create one?

Reply:
yes. 3.0 is not exist. @mahjonp PTAL (It builds in Mac mini in beijin office)

Reply:
Done, now we can download darwin 3.0 tidb-server using URL: https://download.pingcap.org/tidb-v3.0-darwin-amd64.tar.gz

Reply:
@mahjonp @zhouqiang-cl actually, Morgan was asking for tidb-release-3.0-darwin-amd64.tar.gz (only tidb-server) so the user doesn't have to download all components when they're not needed. Could you please also add that package?

Reply:
Also, it looks like the tidb-v3.0-darwin-amd64.tar.gz package is still at version 3.0.0, even though 3.0.1 has been released.